index.html

```diff
@@ -0,0 +1,177 @@
+                <button class="header__button">Download FREE!</button>
+            </article>
+            <div class="screen__container cell">
+                <img src="./images/screen.png" alt="Screen" class="header__screen">
+            </div>
+        </div>
+    </header>
+
+    <section id="features-section" class="feautres row">
+        <div class="feautres__container container">
+            <h1 class="features__text">FEATURES</h1>
+        </div>
+        <div class="features__items container">
+            <article class="features__article cell">
+                <img src="./images/responsive.svg" alt="responsive" class="article__img">
+                <h1 class="article__text">FULLY RESPONSIVE</h1> 
+                <p>Looks amazing on any device:<br>smartphone, tablet, laptop and<br>desktop.</p>
+            </article>
+
+            <article class="features__article cell">
+                <img src="./images/customizable.svg" alt="customizable" class="article__img">
+                <h1 class="article__text">CUSTOMIZABLE</h1> 
+                <p>Change the colors, pictures or<br>any of the sections to suit your<br>needs.</p>
+            </article>
+
+            <article class="features__article cell">
+                <img src="./images/design.svg" alt="design" class="article__img">
+                <h1 class="article__text">SLICK AND BEAUTIFUL<br>DESIGN</h1>
+                <p>Trendy and fresh design, fits<br>any webiste.</p> 
+            </article>
+
+        </div>
+
+
+    </section>
+
+    <section class="sponsors row">
+        <div class="sponsors__container container">
+            <img src="./images/logos.png" alt="logos" class="sponsors__image">
+            
+        </div>
+    </section>
+
+    <section class="description row">
+        <div class="description__container container"> 
+            <div class="description__items cell">
+                <img src="./images/imac.png" alt="imac" class="description__image">
+            </div>
+            <div class="description__items cell">
+                <h1>SIMPLE AND BEAUTIFUL</h1>
+                <p>Use Nova theme for your next web project. It is<br>completely customizable so you can change any of<br>the sections to fit your needs. Nova Theme is Free for<br>any kind of use, personal and commercial. Have fun<br>and good luck!</p>
+            </div>
+        </div>
+    </section>
+
+
+    <section id="pricing-section" class="pricing row">
+        <div class="pricing__text container">
+            <h1>CHOOSE YOUR PRICING PLAN</h1>
+            <p>Pick any of our supper affordable pricing plans</p>
+        </div>
+        <div class="pricing__options container">
+            <article class="pricing__level cell">
+                <h2>BASIC</h2>
+                <h1>$15</h1>
+                <p>per month</p>
+                <h3>Up to 7 Projects<br>2 Additional Developers<br><br><br></h3>
```

Comment:
Tutaj zdecydowanie `ul > li` lista cech danego pakietu.
Używanie `br` dla robienia odstępów to zły pomysł